fields/VideoField.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the YesWiki Extension alternativeupdatej9rem.
+ *
+ * Authors : see README.md file that was distributed with this source code.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace YesWiki\alternativeupdatej9rem\Field;
+
+
+use Psr\Container\ContainerInterface;
+use YesWiki\Bazar\Field\LinkField;
+
+/**
+ * @Field({"video"})
+ */
+class VideoField extends LinkField
+{
+    public function __construct(array $values, ContainerInterface $services)
+    {
+        parent::__construct($values, $services);
+        
+    }
+
+    protected function renderInput($entry)
+    {
+        // Display the linked entries only on update
+    return $this->render("@bazar/inputs/link.twig", [
+            'value' => $this->getValue($entry)
+        ]);
+    }
+
+    protected function renderStatic($entry)
+    {
+        // Display the linked entries only if id_fiche and id_typeannonce
+		$value = $this->getValue($entry);
+		$type = 'peertube'; // type par defaut peertube car le format de l'url ne permet pas de detecter peertube 
+		$instance = 'https://videos.yeswiki.net/';

```

Comment:
```suggestion
		$instance = $this->params->get('default_peertube_server');
```
Je pense qu'il va falloir définir un nouveau paramètre dans le bon fichier `config.yaml` et permettre sa modification dans `{{editonfig}}` (page `?GererConfig`) si cette variable n'y existe pas déjà (Je vais devoir expliquer comment faire ;-))